src/registrar/migrations/0003_alter_userprofile_street1.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.1 on 2022-09-22 16:53
+
+from django.db import migrations, models
```

Comment:
As this is the first PR to include migrations, could I trouble you to squash them? (Just delete and regenerate.) Unless you believe they are logically better represented as three files. That's fine as well.